packages/ods/src/components/breadcrumb/src/components/ods-breadcrumb/ods-breadcrumb.scss

```diff
@@ -0,0 +1,18 @@
+$breadcrumb-spacing: 8px;
+
+:host(.ods-breadcrumb) {
+  display: flex;
+  flex-wrap: wrap;
+  column-gap: $breadcrumb-spacing;
+  align-items: center;
```

Comment:
```suggestion
  align-items: baseline;
  line-height: 1rem;
```
does fix the global alignment with icons:
Before : 
![image](https://github.com/ovh/design-system/assets/5332505/36aef77e-f956-418e-aac0-bf09580bb553)

After :
![image](https://github.com/ovh/design-system/assets/5332505/9f8fd0c2-2989-49b1-8f2a-466ddf243a6b)